src/app/docs/key-info/key-info-docs.component.html

```diff
@@ -0,0 +1,164 @@
+        >
+        </sky-docs-design-guideline-thumbnail>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use definition lists for content that doesn't require user attention."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/key-info/key-info-usage-6.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/guidelines/key-info/key-info-anatomy.png"
+      >
+        <sky-docs-anatomy-item>Value</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Label</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      <sky-docs-design-guideline
+        heading="Layout"
+      >
+        <p>
+          Use the default vertical layout with the label under the value in most situations.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/key-info/vertical-layout.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+      >
+        <p>
+          Use the horizontal layout with the label beside the value for scenarios where vertical space is at a premium. For example, use the horizontal layout to display the total number of items in a list.
+        </p>
+
+      <sky-docs-design-guideline-thumbnail
+        imageSource="~/assets/img/guidelines/key-info/horizontal-layout.png"
+      >
+      </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior & States"
+    >
+      <sky-docs-design-guideline
+        heading="Link styling"
+      >
+        <p>
+          You can style the key info component as a link when you want to let users click the value to navigate to a different location.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+        heading="Components"
+      >
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/components/definition-list">Definition list</a></li>
+        </ul>
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/components/page-summary">Page summary</a></li>
+        </ul>
+     </sky-docs-design-guideline>
+
+     <sky-docs-design-guideline
+      heading="Guidelines"
+    >
+      <ul>
+        <li><a href="https://developer.blackbaud.com/skyux/design/guidelines/call-out-info">Call out information</a></li>
+      </ul>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
```

Comment:
The Development tab content is missing. We need to add JSDoc comments. We need to copy the content for the `layout` property from the existing Development tab and paste it as JSDoc comments https://github.com/blackbaud/skyux-indicators/blob/master/src/app/public/modules/key-info/key-info.component.ts. If you need an example, see the JSDoc comments for dropover at https://github.com/blackbaud/skyux-popovers/blob/master/src/app/public/modules/dropdown/dropdown.component.ts.

In addition, we need to add the description for the `label` and `value` properties (which look like they are currently documented in the wrong places) to https://github.com/blackbaud/skyux-indicators/blob/master/src/app/public/modules/key-info/key-info-label.component.ts and https://github.com/blackbaud/skyux-indicators/blob/master/src/app/public/modules/key-info/key-info-value.component.ts. I think for these items, we need to add the JSDoc comments before the `@Component` line, similar to what we do for the `@Pipe` in the fuzzy date pipe at https://github.com/blackbaud/skyux-datetime/blob/master/src/app/public/modules/date-pipe/fuzzy-date.pipe.ts. But you might need to play around with that and/or ask Steve for some direction.